demo/cards/article/playground

```diff
@@ -0,0 +1,18 @@
+return (
+  <div style={{ width: 360 }}>
+    <CardArticle
+      link='https://github.com/SUI-Components/sui-components'
+      media={{ src: 'http://lorempicsum.com/simpsons/360/180/3' }}
+      tag={{
+        url: 'http://www.coches.net/novedades/',
+        text: 'News'
+      }}
+      comments={{
+        url: 'http://www.coches.net//opiniones/nuevo-hyundai-i10-presentacion',
```

Comment:
http://www.coches.net/opiniones/nuevo-hyundai-i10-presentacion
instead of :)
http://www.coches.net//opiniones/nuevo-hyundai-i10-presentacion